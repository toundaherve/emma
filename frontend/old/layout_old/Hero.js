import React from "react";
import styled from "styled-components";
import theme from "../theme";

import Typography from "../components/Typography";
import Input from "../components/Input";
import Button from "../components/Button";

const OuterWrapper = styled.div`
  height: calc(100vh - 4.5rem);
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: #cccccc;

  & > * {
    width: 20.5rem;
  }
`;

const InnerWrapper = styled.div`
  display: flex;
  flex-direction: column;
  background-color: transparent;

  & > * {
    flex-basis: 100%;
  }

  & > h1 {
    margin-bottom: ${(props) => (props.noMarginBottom ? 0 : theme.spacing[6])};
  }

  & > input {
    margin-bottom: ${theme.spacing[7]};
  }
`;

const Hero = () => {
  return (
    <OuterWrapper>
      <InnerWrapper>
        <Typography variant="h1" align="center">
          Fresh and Tasty homemade food
        </Typography>
        <Input placeholder="Enter postcode" />
        <Button>Order Now</Button>
      </InnerWrapper>
    </OuterWrapper>
  );
};

export default Hero;
