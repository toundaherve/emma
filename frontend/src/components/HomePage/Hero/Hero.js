import React from "react";
import { Link } from "react-router-dom";

const Hero = () => {
  return (
    <div className="hero">
      <div className="masthead">
        <div className="masthead-container">
          <picture>
            {/*<!--[if IE 9]>*/}
            <video style={{ display: "none" }}>
              {/*<![endif]--></video>*/}
              <source
                media="(max-width: 479px)"
                srcSet="https://images.prismic.io/brinker-chilis/332f213b-0908-4491-84da-38da77c2f269_Chili%27s+To+Go+at+home+1.jpg?auto=compress,format&amp;rect=216,0,768,800&amp;w=480&amp;h=500"
              />
              {/*<!--[if IE 9]>*/}
            </video>
            {/*<![endif]--> */}
            <img
              className="background-image-blur image-cover masthead-image b-lazy unblur b-loaded"
              alt="Chili's To Go "
              src="https://images.prismic.io/brinker-chilis/332f213b-0908-4491-84da-38da77c2f269_Chili%27s+To+Go+at+home+1.jpg?auto=compress,format&amp;rect=0,208,1200,384&amp;w=1600&amp;h=512"
            />
          </picture>
          <div className="banner-content hero-content bg-overlay-light">
            <h1
              className="headline-container headline-container-hero center"
              style={{ color: "#F5E9DA" }}
            >
              <span className="subheader top header-shadow">
                Fresh and Homemade
              </span>
              <span className="header header-shadow">Cameroonian food</span>
              <span className="subheader header-shadow">Pickup - Delivery</span>
            </h1>
            <div className="btn-container cta-component center">
              <Link to="/menu" id="home-hero-button-1" className="btn">
                Start Order
              </Link>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default Hero;
