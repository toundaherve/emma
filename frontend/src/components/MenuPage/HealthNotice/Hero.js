import React from "react";

const Hero = () => {
  return (
    <div className="hero">
      <div className="masthead">
        <div className="masthead-container">
          <picture>
            {/*<!--[if IE 9]>*/}
            <video style={{ display: "none" }}>
              {/*<![endif]--></video>*/}
              <source
                media="(max-width: 479px)"
                srcSet="https://images.prismic.io/brinker-chilis/332f213b-0908-4491-84da-38da77c2f269_Chili%27s+To+Go+at+home+1.jpg?auto=compress,format&amp;rect=216,0,768,800&amp;w=480&amp;h=500"
              />
              {/*<!--[if IE 9]>*/}
            </video>
            {/*<![endif]--> */}
            <img
              className="background-image-blur image-cover masthead-image b-lazy unblur b-loaded"
              alt="Chili's To Go "
              src="https://images.prismic.io/brinker-chilis/332f213b-0908-4491-84da-38da77c2f269_Chili%27s+To+Go+at+home+1.jpg?auto=compress,format&amp;rect=0,208,1200,384&amp;w=1600&amp;h=512"
            />
          </picture>
          <div className="banner-content hero-content bg-overlay-light">
            <h1
              className="headline-container headline-container-hero center"
              style={{ color: "#F5E9DA" }}
            >
              <span className="subheader top header-shadow">
                Chili's To Go for your
              </span>
              <span className="header header-shadow">Go-To Crew</span>
              <span className="subheader header-shadow">
                Curbside - Pickup - Delivery
              </span>
            </h1>
            <div className="btn-container cta-component center">
              <a id="home-hero-button-1" className="btn" href="/menu">
                Start Order
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default Hero;
