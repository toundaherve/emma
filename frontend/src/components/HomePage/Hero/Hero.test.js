import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";

import Hero from "./Hero";

it("renders correctly", () => {
  const component = renderer.create(
    <Router>
      <Hero />
    </Router>
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
