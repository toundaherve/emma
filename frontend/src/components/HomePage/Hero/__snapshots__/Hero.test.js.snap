// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="hero"
>
  <div
    className="masthead"
  >
    <div
      className="masthead-container"
    >
      <picture>
        <video
          style={
            Object {
              "display": "none",
            }
          }
        >
          <source
            media="(max-width: 479px)"
            srcSet="https://images.prismic.io/brinker-chilis/332f213b-0908-4491-84da-38da77c2f269_Chili%27s+To+Go+at+home+1.jpg?auto=compress,format&rect=216,0,768,800&w=480&h=500"
          />
        </video>
        <img
          alt="Chili's To Go "
          className="background-image-blur image-cover masthead-image b-lazy unblur b-loaded"
          src="https://images.prismic.io/brinker-chilis/332f213b-0908-4491-84da-38da77c2f269_Chili%27s+To+Go+at+home+1.jpg?auto=compress,format&rect=0,208,1200,384&w=1600&h=512"
        />
      </picture>
      <div
        className="banner-content hero-content bg-overlay-light"
      >
        <h1
          className="headline-container headline-container-hero center"
          style={
            Object {
              "color": "#F5E9DA",
            }
          }
        >
          <span
            className="subheader top header-shadow"
          >
            Fresh and Homemade
          </span>
          <span
            className="header header-shadow"
          >
            Cameroonian food
          </span>
          <span
            className="subheader header-shadow"
          >
            Pickup - Delivery
          </span>
        </h1>
        <div
          className="btn-container cta-component center"
        >
          <a
            className="btn"
            href="/menu"
            id="home-hero-button-1"
            onClick={[Function]}
          >
            Start Order
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
